core/templates/pages/topics-and-skills-dashboard-page/modals/unassign-skill-from-topics-modal.component.html

```diff
@@ -4,14 +4,26 @@ <h3>
   </h3>
 </div>
 <div class="modal-body">
-  <span>This skill is assigned to these following topics, Tick the topics that you want this skill to be unassigned from.</span>
+  <div><em>The topics with this skill as their only diagnostic test skill are not eligible for the unassignment.</em></div><br>
+  <span *ngIf="topicsAssignmentsAreFetched && eligibleTopicsForUnassignment.length !== 0">This skill is assigned to these following topics. Tick the topics that you want this skill to be unassigned from.</span>
```

Comment:
Shouldn't the ngIf be on a checkbox basis though? I.e. we list the topics and show an annotation + disable on a topic-specific basis if a particular topic can't handle the unassignment.

Reply:
Updated based on the discussion